src/mscorlib/src/System.Private.CoreLib.txt

```diff
@@ -678,6 +678,7 @@ ArgumentException_BadMethodImplBody = MethodOverride's body must be from this ty
 Argument_BadParameterCountsForConstructor = Parameter count does not match passed in argument value count.
 Argument_BadParameterTypeForConstructor = Passed in argument value at index {0} does not match the parameter type.
 Argument_BadTypeInCustomAttribute = An invalid type was used as a custom attribute constructor argument, field or property.
+Argument_ConstantField = A custom attribute field was constant, and so is not writable. 
```

Comment:
The existing verbiage for read-only properties is "Argument_NotAWritableProperty = Not a writable property.". Should follow the same pattern.